handout/redux/README.md

```diff
@@ -10,12 +10,14 @@ Where Flux applications traditionally have multiple stores, Redux applications
 have only one global, read-only application state. This state is calculated by
 "reducing" over a collection or stream of actions that update it in controlled ways.
 
-The Redux pattern has become very popular and has inspired
-[Ngrx](https://github.com/ngrx "ngrx collection") which is exclusive to
-Angular2/RxJS.
+One popular Angular 2 specific implementation of the Redux pattern is [Ng2-Redux](https://github.com/wbuchwalter/ng2-redux), which we'll be using to describe how to use this approach in an application.
 
-In this course we discuss a more traditional Redux pattern using the
-[Ng2-Redux](https://github.com/wbuchwalter/ng2-redux) bindings library.
+
+## What is Ngrx?
+
+Redux implementation has been very well received and has inspired the creation of [ngrx](https://github.com/ngrx "ngrx collection"), a set of modules that implement the same way of managing state as well as some of the middleware and tools in the Redux ecosystem. Ngrx was created to be used specifically with Angular 2 and [RxJS](https://github.com/Reactive-Extensions/RxJS), as it leans heavily on the observable paradigm. 
+
+Although we'll be using Ng2-Redux, a lot of the same lessons apply with regards to ngrx though the syntax may be different and have some slight differences in what abstractions are involved.
```

Comment:
Although we will be using ng2-redux, a lot of the same concepts apply in regards to reducers and action creators between redux and ngrx, though the syntax may be different and have some slight differences in what abstractions are involved.